devlab/tests/generate_load_real_env.py

```diff
@@ -0,0 +1,413 @@
+import argparse
+import os
+import time
+import json
+
+from glanceclient import Client as glance
+from novaclient import client as nova
+from neutronclient.neutron import client as neutron
+from keystoneclient.v2_0 import client as keystone
+from cinderclient import client as cinder
+
+import real_env_conf
+
+NOVA_CLIENT_VERSION = real_env_conf.NOVA_CLIENT_VERSION
+GLANCE_CLIENT_VERSION = real_env_conf.GLANCE_CLIENT_VERSION
+NEUTRON_CLIENT_VERSION = real_env_conf.NEUTRON_CLIENT_VERSION
+CINDER_CLIENT_VERSION = real_env_conf.CINDER_CLIENT_VERSION
+
+
+class Prerequisites():
+    def __init__(self, cloud_prefix='SRC'):
+        self.username = os.environ['%s_OS_USERNAME' % cloud_prefix]
+        self.password = os.environ['%s_OS_PASSWORD' % cloud_prefix]
+        self.tenant = os.environ['%s_OS_TENANT_NAME' % cloud_prefix]
+        self.auth_url = os.environ['%s_OS_AUTH_URL' % cloud_prefix]
+        self.image_endpoint = os.environ['%s_OS_IMAGE_ENDPOINT' % cloud_prefix]
+        self.neutron_endpoint = os.environ['%s_OS_NEUTRON_ENDPOINT'
```

Comment:
glance and neutron endpoints can be retrieved from keystone (unless you want to use specific endpoints)